doc/harmony-smp_installation_guide.md

```diff
@@ -18,7 +18,8 @@ Doc. ID: IG-SMP
  28.04.2022 | 1.7     | Minor changes                                                                                           | Petteri Kivimäki
  22.01.2023 | 1.8     | Update SMP Admin Guide link                                                                             | Petteri Kivimäki
  01.06.2023 | 1.9     | Add more information about allowed characters in certificates                                           | Petteri Kivimäki
- 
+ 31.07.2023 | 1.10    | Updates for SMP version 2.0                                                                             | Jarkko Hyöty
```

Comment:
Please also update the version number on line 3 and maybe update the version to 2.0?

Reply:
Looks like the doc versioning is not in sync with the software versions (AP docs are still on 1.x), so I just fixed the version.